app/src/common/img/search-icon-inline.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="16" height="16" viewBox="0 0 16 16">
```

Comment:
Please use one of svg minifiers before commit.
For example you can compress this icon without loss quality from 400 bytes to 192 bytes, it's almost 50%!
Usually I use https://jakearchibald.github.io/svgomg/ but you can choose any other.